ext/standard/tests/file/file_get_contents_basic.phpt

```diff
@@ -13,13 +13,13 @@ echo "*** Testing the basic functionality of the file_get_contents() function **
 echo "-- Testing with simple valid data file --\n";
 
 
-create_files($file_path, 1, "text", 0755, 100, "w", "file", 1, "byte");
+create_files($file_path, 1, "text", 0755, 100, "w", "file_get_contents_basic", 1, "byte");
 var_dump( file_get_contents($file_path."/file1.tmp") );
 delete_files($file_path, 1);
 
 echo "\n-- Testing with empty file --\n";
 
-create_files($file_path, 1, "empty", 0755, 100, "w", "file", 1, "byte");
+create_files($file_path, 1, "empty", 0755, 100, "w", "file_get_contents_basic", 1, "byte");
 var_dump( file_get_contents($file_path."/file1.tmp") );
```

Comment:
Possible test fix?
```suggestion
var_dump( file_get_contents($file_path."/file_get_contents_basic1.tmp") );
```

Reply:
Oops! Should actually have checked this.

Reply:
Indeed! Thank you @Girgias! Also needed to adjust the `delete_files()` calls.